ios general/ios general/CreateEmployeeViewController.m

```diff
@@ -0,0 +1,60 @@
+#import "CreateEmployeeViewController.h"
+
+@interface CreateEmployeeViewController ()
+
+@end
+
+@implementation CreateEmployeeViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.title = @"Add new Employee";
+    
+    [self.firstNameField addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    [self.lastNameField  addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    [self.salaryField    addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    
+    UIBarButtonItem *saveButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemSave target:self action:@selector(saveNewObject:)];
+    self.navigationItem.rightBarButtonItem = saveButton;
+}
+
+- (void)didReceiveMemoryWarning
+{
+    [super didReceiveMemoryWarning];
+}
+
+-(BOOL)textFieldDidChange:(UITextField *)textField
+{
+    if ([textField.text isEqualToString:@""])
+    {
+        self.statusLabel.text = @"All fields are required";
+        textField.layer.borderColor = [[UIColor colorWithRed:1.00 green:0.34 blue:0.34 alpha:1.0] CGColor]; // red
```

Comment:
For floats always use floating point specifier, so not 1.00, but 1.0f, not 0.34, but 0.34f.